dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImpl.java

```diff
@@ -1152,12 +1009,11 @@ protected Contentlet findContentletForLanguage(long languageId, Identifier ident
 	}
 
     @NotNull
-    private List<Contentlet> getContentletsFromSearchResponse(SearchRequest searchRequest) {
-        
+    private List<Contentlet> getContentletsFromSearchResponse(final SearchRequest searchRequest) {
 
-        SearchHits hits = cachedIndexSearch(searchRequest);
+        final SearchHits hits = cachedIndexSearch(searchRequest);
 
-        List<Contentlet> cons = new ArrayList<>();
+        final List<Contentlet> cons = new ArrayList<>();
```

Comment:
rename to contentlets or contentList or so

Reply:
Done